impeller/display_list/dl_golden_unittests.cc

```diff
@@ -182,6 +183,67 @@ TEST_P(DlGoldenTest, GaussianVsRRectBlurScaledRotated) {
   ASSERT_TRUE(OpenPlaygroundHere(builder.Build()));
 }
 
+TEST_P(DlGoldenTest, FastVsGeneralGaussianMaskBlur) {
+  DisplayListBuilder builder;
+  builder.Scale(GetContentScale().x, GetContentScale().y);
+  builder.DrawColor(DlColor::kWhite(), DlBlendMode::kSrc);
+
+  auto blur_sigmas = std::array{5.0f, 10.0f, 20.0f};
+  auto blur_colors = std::array{
+      DlColor::kBlue(),
+      DlColor::kGreen(),
+      DlColor::kMaroon(),
+  };
+
+  auto make_rrect_path = [](const SkRect& rect, DlScalar rx, DlScalar ry) {
```

Comment:
Can you add an explicit return type here please?  It's a bit of a hunt to understand the signature of this closure.

https://google.github.io/styleguide/cppguide.html#Type_deduction